report/3.Theory/1.TrackingObjects.tex

```diff
@@ -0,0 +1,25 @@
+\section{Predicting a moving objects location}
+In order to predict a moving objects location a keen observation to be made is whether the object is flying i.e. if the object is following a ballistic trajectory or whether the object is driving i.e. following a linear trajectory. 
+In the following section we will elaborate on these distinctions in order to gain a deeper understanding on the subject and to be able to complete our objectivs. 
+\subsection{Linear trajectory}
+Given that the target follows a linear trajectory a simple method to calculate the new location of the target can be applied. 
+By using the image processing method as descibed in Section~\ref{} locating the target is considered irrelevent for this subsection. 
+In order to predict our targets location we first need to understand some basic physics of kinematics. 
+That is given an object is traveling with constant speed, the distance traveled by the object can be described as follows:
+\begin{equation}
+s = v \cdot t
+\end{equation}
+where $ s $ is the distance traveled, $ v $ is the velocity and $ t $ is the time passed.
+Hence an object traveling with $ 1 m/s $ for $ 1 s $ will have traveled 1 meter. 
+We can thereby calculate the speed of the target and use image reginition to establish the direction calculate the future position of the target. 
```

Comment:
Something seems odd in the end of this sentence, maybe all it needs is an "and" in between direction and calculate?